### PR TITLE
131408 - change text and insert oasClawback before deferral

### PR DESCRIPTION
**Pull request by MarcoGoC**
### Description
- Change french and english text as requested
- Change the position of the clawback message before the deferral message and the deferral table

#### List of proposed changes:
- as above

### CODE REVIEW

File: i18n/api/fr.ts [hunk 1]
```diff
@@ -387,9 +387,9 @@ const fr: Translations = {
     expectToReceive:
       'Vous pouvez vous attendre à recevoir environ {ENTITLEMENT_AMOUNT_FOR_BENEFIT} par mois.',
     oasClawbackInCanada:
-      "Puisque votre revenu est plus grand que {OAS_RECOVERY_TAX_CUTOFF}, vous devrez rembourser une partie ou la totalité de votre pension de la Sécurité de la vieillesse en raison de l'{LINK_RECOVERY_TAX}.",
+      "Puisque votre revenu est plus grand que {OAS_RECOVERY_TAX_CUTOFF}, vous ne recevrez pas une une partie ou la totalité de votre pension de la Sécurité de la vieillesse en raison de l'{LINK_RECOVERY_TAX}.",
```
Comment (Maxim-Lam) [inline i18n/api/fr.ts]: There is an extra "une" in here

File: i18n/api/fr.ts [hunk 2]
```diff
@@ -387,9 +387,9 @@ const fr: Translations = {
...
     oasClawbackNotInCanada:
-      "Puisque votre revenu est plus grand que {OAS_RECOVERY_TAX_CUTOFF} et que vous vivez à l'extérieur du Canada, vous devrez rembourser une partie ou la totalité de votre pension de la Sécurité de la vieillesse en raison de&nbsp;: <ul class='list-disc' style='padding-left: 24px;'><li style='padding-left: 2px;'>l'{LINK_RECOVERY_TAX};</li><li style='padding-left: 2px;'>l'{LINK_NON_RESIDENT_TAX}.</li></ul></div>",
+      "Puisque votre revenu est plus grand que {OAS_RECOVERY_TAX_CUTOFF} et que vous vivez à l'extérieur du Canada, vous ne recevrez pas une une partie ou la totalité de votre pension de la Sécurité de la vieillesse en raison de&nbsp;: <ul class='list-disc' style='padding-left: 24px;'><li style='padding-left: 2px;'>l'{LINK_RECOVERY_TAX};</li><li style='padding-left: 2px;'>l'{LINK_NON_RESIDENT_TAX}.</li></ul></div>",
```
Comment (Maxim-Lam) [inline i18n/api/fr.ts]: here as well

**Review by Maxim-Lam — CHANGES_REQUESTED** [review]

**Review by Maxim-Lam — APPROVED** [review]
Good once changes implemented

File: i18n/api/fr.ts (hunk 1 above)
Comment (MarcoGoC) [reply i18n/api/fr.ts]: good catch

**Pull request merged**